test/udev_mapping_test.rb

```diff
@@ -17,7 +18,27 @@
       target_map_stub("storage_ppc.yaml")
     end
 
-    it "return argument for non-udev mapped device names" do
+    it "returns mapped raid name for partitioned devices" do
+      allow(Yast::Storage).to receive(:GetContVolInfo) do |_dev, info|
+        info.value["vdevice"] = "/dev/md126p1"
+        info.value["cdevice"] = ""
+        true
+      end
+
+      expect(subject.to_kernel_device("/dev/md/crazy_name")).to eq "/dev/md126p1"
```

Comment:
This test looks strange to me since the return value of GetContVolInfo is independent of its input.